azavea/pwd-fishway
# Add video controls to See The Fishway and Quiz components

## Overview
Add video controls to See The Fishway and Quiz components

These controls are encapsulated in a new VideoPlayer component that bundles together the video, play/pause button and scrubber.

### Demo

![Peek 2019-05-24 15-19](https://user-images.githubusercontent.com/4432106/58351989-46a50900-7e38-11e9-95c4-e17302d01938.gif)
![Peek 2019-05-24 15-20](https://user-images.githubusercontent.com/4432106/58351990-46a50900-7e38-11e9-8aab-65f658f52d64.gif)


### Notes

I used the [`rc-slider`](https://github.com/react-component/slider) library for the slider component, because it seemed well supported and fits our needs.


## Testing Instructions

 * `scripts/update`
 * `scripts/server`
 * Videos should continue to autoplay on See The Fishway and Test your Skills tabs
 * Pause button should change to a play button when the video is paused and when it ends
 * The scrubber position should update as the video plays
 * Changing the scubber position should change the video position, and the video should continue playing afterwards, unless the scrubber position is at the end of the slider
